CONTRIBUTING.md

```diff
@@ -134,6 +134,17 @@ wget https://github.com/matterport/Mask_RCNN/releases/download/v2.0/mask_rcnn_co
     export AUTO_SEGMENTATION_PATH="/path/to/dir" # dir must contain mask_rcnn_coco.h5 file
 ```
 
+### Tracking
+- Download the GOTURN model if the the OpenCV GOTURN tracker is to be used.
+```sh
+#This downloads it into the cvat folder
+components/tracking/install.sh
```

Comment:
> into the cvat folder

Root directory doesn't look as a right place.
I believe it make sense to download it into appropriate places, like`components/tracking/model`.

Or take a look in a direction (investigate) of using docker volumes / mounted shares with large models.